lib/rules/no-tabs.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @fileoverview Rule to check for tabs inside a file
+ * @author Gyandeep Singh
+ */
+
+"use strict";
```

Comment:
http://eslint.org/docs/developer-guide/code-conventions#file-format

Just missing the "Public Interface" separator.